handlers/deploy.go

```diff
@@ -118,16 +116,19 @@ func MakeDeployHandler(functionNamespace string, clientset *kubernetes.Clientset
 
 func makeDeploymentSpec(request requests.CreateFunctionRequest, existingSecrets map[string]*apiv1.Secret, config *DeployHandlerConfig) (*v1beta1.Deployment, error) {
 	envVars := buildEnvVars(&request)
-	path := filepath.Join(os.TempDir(), ".lock")
 	probe := &apiv1.Probe{
 		Handler: apiv1.Handler{
-			Exec: &apiv1.ExecAction{
-				Command: []string{"cat", path},
+			HTTPGet: &apiv1.HTTPGetAction{
+				Path: "/healthz",
+				Port: intstr.IntOrString{
+					Type:   intstr.Int,
+					IntVal: int32(watchdogPort),
+				},
 			},
 		},
-		InitialDelaySeconds: 3,
+		InitialDelaySeconds: 0,
```

Comment:
Please introduce a configuration item for this. You can largely copy and paste from the existing variables.

Should also be available via helm as an option.

Reply:
ok